### PR TITLE
AC: update accuracy evaluation method for arcface models

### PR DESCRIPTION
**Pull request by eaidova**
updated accuracy measurement according proposed by authors

### CODE REVIEW

File: models/public/face-recognition-resnet34-arcface/accuracy-check.yml [hunk 1]
```diff
@@ -11,14 +11,7 @@ models:
             shape: 3, 112, 112
 
     datasets:
-      - name: lfw
-
-        preprocessing:
-          - type: bgr_to_rgb
```
Comment (Anna-Mironova) [inline models/public/face-recognition-resnet34-arcface/accuracy-check.yml:17]: Should we keep this preprocessing  for mxnet model?

File: models/public/face-recognition-resnet50-arcface/accuracy-check.yml [hunk 2]
```diff
@@ -11,14 +11,7 @@ models:
             shape: 3, 112, 112
 
     datasets:
-      - name: lfw
-
-        preprocessing:
-          - type: bgr_to_rgb
```
Comment (Anna-Mironova) [inline models/public/face-recognition-resnet50-arcface/accuracy-check.yml:17]: Should we keep this preprocessing  for mxnet model?

File: models/public/face-recognition-resnet34-arcface/accuracy-check.yml (hunk 1 above)
Comment (eaidova) [reply models/public/face-recognition-resnet34-arcface/accuracy-check.yml:17]: updated

File: models/public/face-recognition-resnet50-arcface/accuracy-check.yml (hunk 2 above)
Comment (eaidova) [reply models/public/face-recognition-resnet50-arcface/accuracy-check.yml:17]: updated

**Review by Anna-Mironova — APPROVED** [review]

**Pull request merged**